data/tlg0540/tlg008/tlg0540.tlg008.perseus-eng2.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
```

Comment:
I haven't decided if we should permanently change this or not. I wish we could get answers from the stoa people.

Reply:
Ideally this would not be inconsistent in the collections.

Reply:
I had actually changed it for the moment, simply so that I could validate the file.

Reply:
No, I agree with that. I'm just wondering how to address this going forwards. I'm using it, too, so I already have a split corpus. I'm tired of bothering them to fix it.